v19.2/managed-console-access-management.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Console Access Management
+summary: Manage your account roles and permissions.
+toc: true
+build_for: [managed]
+---
+
+The Access page displays the name, email address, role, and invite acceptance status of the Team Members with access to your CockroachDB Cloud cluster. To view the Access page, [sign in to the Console](managed-sign-up-for-a-cluster.html#sign-in) and on the **Cluster Overview** page, click **Access**.
+
+<Screenshot here>
+
+## Roles
+
+Every CockroachDB Cloud user is either a Developer or a Console Admin.
+
+### Developer
+
+A Developer is a limited-access role. A Developer cannot invite Team Members to the Console or access databases.
+
+To access a database, you need to ask a Console Admin for details of a SQL user. To find out who's your Console Admin, check the Access page.
+
+### Console Admin
+
+A Console Admin is an all-access role. A Console Admin can perform the following tasks:
+
+- [Invite Team Members to CockroachDB Cloud]()
```

Comment:
There are no links here. Does that get automatically changed to in-page anchors jumps?